filesystems/drs/src/test/scala/cromwell/filesystems/drs/DrsResolverSpec.scala

```diff
@@ -14,6 +14,7 @@ class DrsResolverSpec extends AnyFlatSpec with CromwellTimeoutSpec with Matchers
   private val marthaConfig: Config = ConfigFactory.parseMap(
     Map(
       "martha.url" -> "https://martha-url/martha_v3",
+      "access-token-acceptable-ttl" -> "1 minute"
```

Comment:
Does this mean we have a new config variable or did this just need to be added here for the tests?

Reply:
Not a new variable, just changing the way we access the existing variable in such a way that it needed to be present in this mock config.